public/css/user/helpOthers.css

```diff
@@ -100,6 +100,10 @@
   padding-top: 0;
   color: #333;
 }
+.user-restrictions{
+  color: #333;
+  font-family: rubik;
+}
 .error {
   color: rgb(107, 0, 0);
   font-family: roboto;
```

Comment:
u can delete font family Roboto and put it for html or body tag
u repeat it 11 times